GooglePlayInstant/LoadingScreen/LoadingScreenScript.cs

```diff
@@ -31,15 +35,20 @@ private IEnumerator Start()
 
             if (loadingScreenConfigJsonTextAsset == null)
             {
-                throw new FileNotFoundException(
-                    "Error on Loading Screen: Could not find LoadingScreenConfig.json file in Resources folder.");
+                throw new FileNotFoundException("LoadingScreenConfig.json missing in Resources folder.");
             }
 
-            var loadingScreenConfigJson = loadingScreenConfigJsonTextAsset.ToString();
+            var loadingScreenConfig = loadingScreenConfigJsonTextAsset.ToString();
 
-            var loadingScreenConfigJsonObj = JsonUtility.FromJson<LoadingScreenConfig>(loadingScreenConfigJson);
+            var loadingScreenConfigJsonObj = JsonUtility.FromJson<LoadingScreenConfig>(loadingScreenConfig);
 
             yield return StartCoroutine(GetAssetBundle(loadingScreenConfigJsonObj.assetBundleUrl));
+
+            if (_bundle == null)
+            {
+                Debug.LogError("AssetBundle failed to be downloaded.");
```

Comment:
You need to return here or you'll get an exception below

Reply:
Resolved.